File: sorting/selectionsort/java/selectionsort.java
import java.util.*;
class Select
{
    public static void main()
    {
        Scanner sc=new Scanner(System.in);
        int size;
        int temp;
        System.out.println("Enter the size of the array"); 
        size=sc.nextInt();
        int arr[]=new int[size];
        System.out.println("Enter array elements");
        for(int i=0;i<size;i++)
        {
            arr[i]=sc.nextInt();
            
        }
        for(int k=0;k<size;k++)
        {
            for(int j=k+1;j<size;j++)
            {
                if(arr[k]>arr[j])
                {
                    temp=arr[j];
                    arr[j]=arr[k];
                    arr[k]=temp;
                }
            }
        }
        System.out.println("Sorted array \n");
        for(int g=0;g<size;g++)
        {
            System.out.println(arr[g]);
        }
    }
}

File: Sorting/selection_sort/C++/selection_sort.cpp
#include<iostream.h>
#include<conio.h>
void main()
{
	clrscr();
	int size, arr[50], i, j, temp,pos;
	cout<<"Enter Array Size : ";
	cin>>size;
	cout<<"Enter Array Elements : ";
	for(i = 0; i < size; i++)
	{
		cin>>arr[i];
	}
	cout<<"Sorting array using selection sort...\n";
    	for(i = 0; i < size; i++)
    	{
        	pos = i;
        	for(j = i + 1 ; j < size  ; j++)
        	{
            	if(arr[j] < arr[pos])
                	pos = j;
        	}
        	temp = arr[i];
        	arr[i] = arr[pos];
        	arr[pos] = temp;
    	}
	cout<<"Now the Array after sorting is :\n";
	for(i = 0; i < size; i++)
	{
		cout<<arr[i]<<" ";
	}
	getch();
}
